setup/monitoring/index.md

```diff
@@ -6,3 +6,262 @@ sidebar:
+sudo apt-get install spinnaker-monitoring-third-party -y
+```
+
+  2. [Configure the Spinnaker Monitoring Daemon](#configure-the-spinnaker-monitoring-daemon)
+
+  3. Install the 3rd-party monitoring package
+     * [Configuring Datadog](/setup/monitoring/datadog)
+     * [Configuring Prometheus](/setup/monitoring/prometheus)
+     * [Configuring Stackdriver](/setup/monitoring/stackdriver)
+
+  4. Restart the Spinnaker Monitoring Daemon
+```
+sudo service spinnaker-monitoring restart
+```
+
+# Configuring the Spinnaker Monitoring Daemon
+
+## Consuming Metrics
+
+Spinnaker publishes internal metrics using a multi-dimensional data model
+based on "tags". Each "metric" has a name and type. Each data point is a
+numeric value that is time-stamped at the time of reporting and tagged with
+a set of one or more "label"="value" tags. These tag values are strings,
+though some may have numeric-looking values. Taken together, the set of
+tags convey the context for the reported measurement. Each of these
+contexts forms a distinct time-series data stream.
+
+For example a metric counting calls may be tagged with a "status" label
+and values indicating whether the call was successful or not. So rather
+than having two metrics, one for successful calls and the other for
+unsuccessful calls, there is a single metric, where the underlying
+monitoring system can filter the successful from unsuccessful as you want
+depending on how you wish to abstract and interpret the data. In practice
+the metrics have many tags providing a lot of granularity and ways in
+which you can aggregate and interpret them. The data model is described
+further in [the Monitoring reference section](/reference/monitoring/).
+
+In practice there are relatively few distinct metric names (a few hundred).
+However when considering all the distinct time-series streams from the
+different label values there are several thousand distinct streams. Some
+metrics are tagged with the application or account they were used on
+behalf of, so the number of streams may grow as the scope of your
+deployment grows. Typically you will be aggregating these dimensions
+together while breaking out along others. The granularity can come in
+handy when it comes time to diagnose problems or investigate for deeper
+understanding of runtime behaviors but you can aggregate across dimensions
+(or parts of dimensions) when you dont care about that level of refinement.
+
+Each microservice defines and exports its own metrics. The
+`spinnaker-monitoring` daemon augments these by identifying which
+microservice they came from (how it does so depends on the backend
+metrics system you are using). There are no global "Spinnaker" metrics.
+There are only individual metrics on the individual instance of
+concrete microservices that, taken together, compose a "Spinnaker" deployment.
+
+
+### Types of Metrics
+
+There are two basic types of metrics currently supported,
+*counters* and *gauges*.
```

Comment:
I like that you've defined these out. Maybe in the reference section point back to this page to understand the concepts?